gbfs.md

```diff
@@ -259,7 +261,7 @@ Field Name | REQUIRED | Type | Defines
 
 ### gbfs.json
 
-The `gbfs.json` discovery file SHOULD represent a single system or geographic area in which vehicles are operated. The location (URL) of the `gbfs.json` file SHOULD be made available to the public using the specification’s [auto-discovery](#auto-discovery) function.<br />The following fields are all attributes within the main `data` object for this feed.
+The `gbfs.json` discovery file SHOULD represent a single system or geographic area in which vehicles are operated. The location (URL) of the `gbfs.json` file SHOULD be made available to the public using the specification’s [auto-discovery](#auto-discovery) function. To avoild circular references, this file MUST NOT contain links to `manifest.json` files.<br />The following fields are all attributes within the main `data` object for this feed.
```

Comment:
Mostly just curious: you call out avoid linking to `manifest.json` to avoid circular dependencies, but `system_information.json` contains a link to the manifest url, providing a circular link of sorts (`gbfs.json -> system_information.json -> manifest.json`).  Is the idea here that consumers are automatically downloading all files referenced in `gbfs.json` without regard to type and you are worried about them getting stuck in a download loop?  Does a similar loop exist for `gbfs_versions.json` and has it been a problem?

Reply:
> Is the idea here that consumers are automatically downloading all files referenced in gbfs.json without regard to type and you are worried about them getting stuck in a download loop?

Yes - that's  exactly the concern. It's true that looping is also possible via `system_information.json` but the `manifest.json` link needs to go someplace and that's the logical location for it. We haven't heard of any one having problems with `gbfs_versions.json` . This issue was specifically [called out](https://docs.google.com/document/d/1jAwKY9e7rz3Do6EkUecR59Oj5Nm5bM3isNN3plvSE3s/edit#heading=h.70le483tkff0) when I started work on this endpoint.